### DISCUSSION TITLE
NER model architecture

### DISCUSSION THREAD

**Discussion by sonynavdeep81** [explosion/spaCy#10586] [Help: Model Advice] [answered]
I have built a named-entity recognizer and getting good results on one of the asian languages. Now I want to publish my results in a reputed journal and for the same I need to explain the readers about the spaCy's NER model/architecture but I am not able to find it anywhere. Its surprising (as spaCy is open-source), may be I missed it. The only thing related to it I found is [https://explosion.ai/blog/parsing-english-in-python](url) but it no way shows the NER model architecture. Please guide me . Thanks.

**Comment by adrianeboyd** [comment, marked as answer]
Hi, this video is a good general intro to the NER architecture: https://www.youtube.com/watch?v=sqDHBH9IjRU. It's for spacy v2, but all the basics of the NER model are the same in v3.

An updated link to the slides: https://raw.githubusercontent.com/explosion/talks/master/2017-11-02_Practical-and-Effective-Neural-NER.pdf

**Reply by sonynavdeep81** [reply]
Thanks a lot!
